### PR TITLE
Use REDIS_URL var for Action Cable config template

### PR DESCRIPTION
**Pull request by ghost**
### Summary

Use environment variables in the action cable configuration, using the common `REDIS_URL` to
configure it, and if is not set, fallback to the default configuration (localhost).

This doesn't fix any bugs, however, today we had a small downtime because we missed the `Action Cable` configuration, I found this tweak useful.

### CODE REVIEW

File: railties/lib/rails/generators/rails/app/templates/config/cable.yml [hunk 1]
```diff
@@ -6,5 +6,5 @@ test:
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: <%%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```
Comment (palkan) [inline railties/lib/rails/generators/rails/app/templates/config/cable.yml]: `<%=`?

Comment (MrOutis) [inline railties/lib/rails/generators/rails/app/templates/config/cable.yml]: I took as guide the other templates, @palkan, see the following: [postgresql.yml](https://github.com/MrOutis/rails/blob/e267db2914785e20bee17097247dc84539ea4c39/railties/lib/rails/generators/rails/app/templates/config/databases/postgresql.yml)

Honestly, I don't know how it works, but I think is kind of "escaping" the ERB interpolation.

Comment (palkan) [inline railties/lib/rails/generators/rails/app/templates/config/cable.yml]: Oh, right, we're inside a template which is ERB-processed too. Sorry, everything is correct)

That's the way it works–[link](https://docs.puppet.com/puppet/5.0/lang_template_erb.html#literal-tag-delimiters).

Comment (MrOutis) [inline railties/lib/rails/generators/rails/app/templates/config/cable.yml]: :)

**Pull request merged**